src/main/scala/akka/minion/Dashboard.scala

```diff
@@ -11,24 +13,51 @@ object Dashboard {
   def props(settings: Settings): Props = Props(new Dashboard(settings))
 
   // Main dashboard
+  trait Action
+  object Action {
+    case object Commented extends Action
+    case object Approved extends Action
+    case object RequestedChanges extends Action
+    case object OpenedPr extends Action
+
+    def toAction(review: PullRequestReview): Action = review match {
+      case r if r.commented => Action.Commented
+      case r if r.approved => Action.Approved
+      case r if r.changesRequested => Action.RequestedChanges
+    }
+  }
+
+  trait PrValidationStatus
+  object PrValidationStatus {
+    case object Success extends PrValidationStatus
+    case object Pending extends PrValidationStatus
+    case object Failure extends PrValidationStatus
+  }
+
+  case class Person(login: String, avatarUrl: String)
+  case class Performance(person: Person, action: Action, performedAt: ZonedDateTime)
```

Comment:
Yea. Late evening bad naming decision. I first thought of it as a performance of an actor. But now I clearly see the weirdness that this word brings.